src/snipe/cli/cli_ops.py

```
@@ -989,85 +1088,101 @@ def detect_delimiter(file_path):
+    total_experiments = len(results)
+    successful_experiments = 0
+    for r in results:
+        if r['status'] == 'success':
+            successful_experiments += 1
+    failed_experiments = total_experiments - successful_experiments
+    # total_skipped = sum(len(r.get('skipped_signatures', [])) for r in results)
+    total_skipped = 0
+    for r in results:
+        total_skipped += len(r.get('skipped_signatures', []))
+
+    click.echo(f"\nGuided Merge Summary:")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)